spark-indexer/src/main/java/org/uniprot/store/spark/indexer/common/store/DataStoreParameter.java

```diff
@@ -0,0 +1,23 @@
+package org.uniprot.store.spark.indexer.common.store;
+
+import java.io.Serializable;
+
+import lombok.Builder;
+import lombok.Getter;
+
+/**
+ * @author lgonzales
+ * @since 02/12/2020
+ */
+@Getter
+@Builder
```

Comment:
good refactoring. 👍  it'd be better if you can put a small description about this class